src/web/assets/cp/dist/js/Craft.js

```diff
@@ -12652,8 +12677,47 @@ Craft.EditableTable = Garnish.Base.extend(
                 this.addListener(Garnish.$win, 'resize', 'initializeIfVisible');
             }
         },
+        updateAddRowButton: function() {
+            if ((this.hasMaxRows && this.rowCount >= this.maxRows) ||
+                (this.hasMinRows && this.rowCount < this.minRows)) {
+                this.$addRowBtn.css('opacity', '0.2');
+            } else {
+                this.$addRowBtn.css('opacity', '1');
+            }
```

Comment:
I know the button will no longer add or remove rows if it is greyed out, but other than greying out the button, we will want to also turn off the cursor hand pointer so users no longer think it is clickable.